Doc/library/importlib.metadata.rst

```diff
@@ -13,21 +13,39 @@
 
 **Source code:** :source:`Lib/importlib/metadata/__init__.py`
 
-``importlib.metadata`` is a library that provides access to installed
-package metadata, such as its entry points or its
-top-level name.  Built in part on Python's import system, this library
+``importlib_metadata`` is a library that provides access to
+the metadata of an installed :term:`packaging:Distribution Package`,
+such as its entry points
+or its top-level names (:term:`packaging:Import Package`s, modules, if any).
```

Comment:
The immediate error that you need to escape alphanumeric characters after ending backticks, like this:

```suggestion
or its top-level names (:term:`packaging:Import Package`\s, modules, if any).
```

Or like this

```suggestion
or its top-level names (:term:`packaging:Import Package`\ s, modules, if any).
```

which is what the error message is talking about. However, there's a bigger issue in that the Intersphinx references won't work unless we add enable Intersphinx and add `packaging` as an Intersphinx target, which is easy to do and I'd like to do, but needs to be discussed first since it could have consequences particularly for downstreams, since it won't work as intended without network access or a local cache when building the docs from source.  We can talk about it at the sprint today and our docs team meeting in a few minutes, either we'll add it or worse we could just convert them to hardcoded links. Gotta run to that but I'll be back in a bit.

Reply:
Well shame on me. I added that bug (with the trailing `s`) to the docs in importlib_metadata, and never confirmed that it rendered properly. I applied the suggested fix at python/importlib_metadata@9674092.